### PR TITLE
Eliminate logging race condition in ActorSystemSpec

### PR DESCRIPTION
**Pull request by johanandren**
Refs #28648

Runaway logging from before the EventFilter would sometimes leak into the expect.

### CODE REVIEW

File: akka-actor-tests/src/test/scala/akka/actor/ActorSystemSpec.scala [hunk 1]
```diff
@@ -181,7 +181,7 @@ class ActorSystemSpec extends AkkaSpec(ActorSystemSpec.config) with ImplicitSend
         a.tell("run", probe.ref)
         probe.expectTerminated(a)
 
-        a.tell("boom", ActorRef.noSender)
+        // Expecting two log entries: one from the actor system at info level and one at warning level from the logging testkit
```
Comment (chbatey) [inline akka-actor-tests/src/test/scala/akka/actor/ActorSystemSpec.scala]: too much boom

**Review by chbatey — APPROVED** [review]
LGTM

**Comment by johanandren** [comment]
PLS BUILD

**Review by raboof — APPROVED** [review]
:boom:

**Pull request merged**